src/lib/components/layouts/Error.svelte

```diff
@@ -8,10 +9,13 @@
   import UserFeedback from "../forms/UserFeedback.svelte";
 
   import { getCurrentUser } from "$lib/utils/permissions";
+  import { APP_URL, SIGN_IN_URL } from "@/config/config";
 
   const me = getCurrentUser();
 
   let feedbackOpen = false;
+
+  $: sign_in_url = new URL(`?next=${APP_URL}`, SIGN_IN_URL);
```

Comment:
I think if we use `$page.url` here instead of `APP_URL`, we can send someone directly back to this page.

```suggestion
  $: sign_in_url = new URL(`?next=${$page.url.href}`, SIGN_IN_URL);
```

Haven't actually tested this yet.

Reply:
I'll push the change and see if it works in preview